shell/utils/create-yaml.js

```diff
@@ -43,14 +43,33 @@ export const NEVER_ADD = [
   'metadata.initializers',
   'metadata.managedFields',
   'metadata.ownerReferences',
-  'metadata.resourceVersion',
+  // 'metadata.resourceVersion',
```

Comment:
how come this was removed?

Reply:
🤷 
Again, I have no idea what is the issue requirement, it can be restored.

Reply:
added back in to this collection. prop path is removed in steve specific context only